src/index.html

```diff
@@ -5,8 +5,247 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="moyo_header">
+      <nav>
+        <div class="header">
+          <a href="#" class="home header__home">
+            <img
+              src="images/logo.png"
+              alt="logo"
+              class="home__img"
+            >
+          </a>
+          <div class="links header__links">
+            <a href="#apple" class="links__link is_active">APPLE</a>
+            <a href="#samsung" class="links__link">SAMSUNG</a>
+            <a href="#s_and_ph" class="links__link">СМАРТФОНИ И ТЕЛЕФОНИ</a>
+            <a
+              href="#lp_and_pc"
+              class="links__link"
+              data-qa="hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+            <a href="#gad" class="links__link">ГАДЖЕТЫ</a>
+            <a href="#pln" class="links__link">ПЛАНШЕТЫ</a>
+            <a href="#photo" class="links__link">ФОТО</a>
+            <a href="#video" class="links__link">ВИДЕО</a>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main class="page">
+      <div class="page__first-block">
+        <div class="product-card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="imac"
+            class="photo product-card__photo"
+          >
+          <h1 class="model-name product-card__model-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+          <section class="model-code product-card__model-code">Код товара: 195434</section>
+          <section class="reviews product-card__reviews">
```

Comment:
Sections are usually used for grouping content that has mutual relations. In my opinion it's semantically wrong to use it on line 54. You can wrap the stars, reviews and price in one section.

Reply:
Yes, `section`  tag should be used only for some independent and sufficent parts of the page.